tests/test.sh

```diff
@@ -61,12 +82,12 @@ function launch_containers() {
   cd notebooks/$engine
   if "$rebuild" = true; then
       echo "Rebuild $engine Containers, as requested"
-      docker-compose down -v
-      docker-compose build
+      $DOCKER_COMPOSE_CMD down -v
```

Comment:
maybe just me stylistically, but I'd rather just see `docker compose` everywhere instead of another variable with indirection ;-)

Reply:
Should probably update the README to use the docker compose.

Reply:
Agreed

Reply:
and get rid of `$DOCKER_COMPOSE_CMD` in favour of `docker compose` ?

Reply:
whelp..  too late...